specification/columns/contractedcost.md

```diff
@@ -2,6 +2,8 @@
 
 Contracted Cost represents the cost calculated by multiplying [*contracted unit price*](#glossary:contracted-unit-price) and the corresponding [Pricing Quantity](#pricingquantity). Contracted Cost is denominated in the [Billing Currency](#billingcurrency) and is commonly used for calculating savings based on negotiation activities, by comparing it with [List Cost](#listcost). If negotiated discounts are not applicable, the Contracted Cost defaults to the List Cost.
 
+When aggregating Contracted Cost for savings calculations, it's important to exclude either one-time or recurring charges ([ChargeCategory](#chargecategory)"Purchase") that are paid to cover future eligible charges or the covered charges themselves. This exclusion helps prevent double counting of these charges in the aggregation. Which set of charges to exclude depends on whether cost are aggregated on a billed basis (exclude covered charges) or accrual basis (exclude Purchases for future charges). For instance, charges categorized as [ChargeCategory](#chargecategory) "Purchase" and their related [ChargeCategory](#chargecategory) "Tax" charges for a [Commitment-Based Discount](#glossary:commitment-based-discount) might be excluded from an accrual basis cost aggregation of Contracted Cost. This is because the "Usage" and "Tax" charge records provided during the term of the commitment discount already specify the Contracted Cost.
```

Comment:
Can you rephrase "billed basis" and "accrual basis" to be more inclusive? Maybe explain the scenario in natural language without jargon and include `(also known as "billed basis")`?

Reply:
In the past we've said terms that are not industry terms and can be easily googled don't need special treatment. We used the same logic when deciding what's needed for the glossary and what can be skipped. Happy to look at a suggestion but my personal view is these are pretty standard terms that a quick Google brings up if anyone isn't sure.

Reply:
Udam will create a new ticket for 1.1 and this will be resolved.